sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -297,6 +306,10 @@ public DataSourceConfiguration withConnectionProperties(
       return builder().setConnectionProperties(connectionProperties).build();
     }
 
+    public DataSourceConfiguration withPoolMaxTotal(int poolMaxTotal) {
```

Comment:
This parameter has no effect, please remove it. A single DoFn instance is used from a single thread, so there will never be more than 1 connection at the same time.